perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/FacilitiesManagerBlImpl.java

```diff
@@ -911,18 +917,82 @@ public void checkFacilityContactExists(PerunSession sess, Facility facility, Str
 	}
 
 	@Override
+	public List<Service> getAssignedServices(PerunSession sess, Facility facility) throws InternalErrorException {
```

Comment:
Yes, I agree, I just blindly copied it from ResourceManager getAssignedResources. I changed it there too.